src/main/java/in/twizmwaz/cardinal/command/JoinCommand.java

```diff
@@ -20,6 +24,12 @@
     @Command(aliases = {"join", "play"}, desc = "Join a team.", usage = "[team]")
     public static void join(final CommandContext cmd, CommandSender sender) throws CommandException {
         if (sender instanceof Player) {
+            Player player = (Player) sender;
```

Comment:
If someone is made a livestreamer while playing a match, aren't they locked into their team?